File: vector.f
c G. Mandaglio oct 17 2016
c simple program twin with a c++ version
      Program vector
      integer indice,i,vettor(100)
     

      print*,"dammi un numero\n"
      read*,indice

c      integer vettor(indice)

      do i=0,indice
      vettor(i) = i+1
      enddo

      do i=0, indice
       print*,vettor(i)
      enddo

      return
      end

File: vector.cpp
// G. Mandaglio oct 17 2016
// simple program twin with a fortran version

// string::operator+=
#include <iostream>
#include <stdio.h>

using namespace std;
int main ()
{
  int indice;
  cout<<"dammi un numero"<<endl;
  cin>>indice;

   int vector[indice];

   for(int i=0;i<indice;i++) vector[i]=i;
   for(int i=0;i<indice;i++) cout<<vector[i]<<endl;

  return 0;
}
